templates/zerver/register.html

```diff
@@ -53,7 +53,7 @@
 
         <div class="input-box full-width">
             <div class="help-box">
-                {{ _('Zulip has full unicode support, so you can spell your name exactly how you want it to appear.') }}
```

Comment:
> Spell your name however you like.


This does not convey any useful information to the use. We should probably just remove this message. The placeholder for this input already have a very good example. Also the box in which the message is shown is way too much big for this message.
![screenshot from 2017-06-24 00 24 56](https://user-images.githubusercontent.com/7190633/27496538-03e1cd74-5874-11e7-9a13-069f9d493822.png)